docs/metasploit-framework.wiki/Metasploit-Guide-Network-Pivoting.md

```diff
@@ -0,0 +1,326 @@
+# Network Pivoting
```

Comment:
What's the main difference between this and https://github.com/rapid7/metasploit-framework/blob/37fe3b909a298e46db21380ebcda7965ad6fe492/docs/metasploit-framework.wiki/Pivoting-in-Metasploit.md ? 👀

Reply:
I didn't know there was some existing documentation about this. I just followed the internal "Docs ideas" document you wrote earlier this year, which is suggesting a "Network Pivoting" documentation under the Pentesting section. I believe something more focus on specific pentesting workflows: initial compromise, internal network reconnaissance, lateral movement, etc.